drafts/developer-guide/pages/data-access-layer/module-config.adoc

```diff
@@ -0,0 +1,38 @@
+include::ROOT:partial$variables.adoc[]
+
+= Compose Module configuration.
```

Comment:
At the end of the document, where you talk about how values are stored (in a JSON, as a separate column, ...); would it make sense to clearly list and describe all of the options (in a table or a list)?

The available options are:

* Column,
* JSON, 
* Alias